float64 ts
EcatSlaveCommand[] slaves